docs/source/sci_api_tutorial/electric_field.rst

```diff
@@ -0,0 +1,122 @@
+==============================================
+Electric Field
+==============================================
+
+
+Briefs
+=========================================================================================
+
+The `electric_field_strength_at_along` function calculates the electric field strength at a specified point between two atoms based on their coordinates and charge, optionally considering a specified direction, and supports different units for the result.
+
+
+Input/Output
+=========================================================================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/ele_field_api.svg
+        :width: 100%
+        :alt: Electric Field
+
+Input
+-----------------------------------------------------------------------------------------
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+
+**Input**: The ``stru``, ``p1``, ``p2``, ``d1``, and ``region_pattern``  are required input
+
+    - ``stru``:
+        Structure which represent the structure of the target enzyme need to obtain from . MUST have charges initialized.
+        
+        .. admonition:: How to obtain
+            
+            | A protein strutcuture object can be obtained by this `APIs <PDBParser.html>`_.
+
+    - ``p1``, ``p2`` and ``d1``:
+        Specify either the direction or the point for measuring the electric field, with combinations like (p1, p2) or (p1, d1), where p1 and p2 represent an Atom() or a Cartesian point. d1 the vector that defines the direction of field strength projection
+        
+        .. admonition:: How to obtain
+            
+            | A p1 and p2 might be `Atom()` object which can be obtained by these `APIs <obtaining_Atom.html>`_. The p1 and p2 can also be the coordinates (x, y, z). 
```

Comment:
We will have a document talking about the Structure class. We wlll link to a section about Atom that page here.

Reply:
give them examples of class methods.
get()
find_atom_...()

Reply:
the name of the file with be structure_class.html